README.md

```diff
@@ -121,3 +121,7 @@ meson setup _build
 ninja -C _build
 ninja install -C _build
 ```
+
+## 👫 Code of Conduct
+This project follows the [GNOME Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct). Please
```

Comment:
Looks like the end of the wiki is getting closer and closer, they have moved the page:

```suggestion
This project follows the [GNOME Code of Conduct](https://conduct.gnome.org/). Please
```

Reply:
Ah nice - the wiki won't be missed